MagaluCloud/magalu
# feat(dbaas): add example to dbaas instances create

## Related Issues

- #688

### Pull request checklist


## How to test it

Run `go run . dbaas instances create -h` and check that the example can be used to create a dbaas instance by copying it and running it
